tests.py

```diff
@@ -196,3 +203,26 @@ def test_valid_hook_module(tmpdir, monkeypatch):
     output = result.output.splitlines()
     assert output[0] == "from on_pre_build"
     assert output[-1] == "from on_post_build"
+
+
+def test_disabling_plugin(tmpdir, monkeypatch):
```

Comment:
this test could use pytest's `parametrize` for `enable=True` and `enable=False` to show the difference

then in `setup_mkdocs` for `enabled` you'd use the argument from `parametrize`, and in asserts something like that:  
```python
assert ("from on_pre_build" in output) == enabled
```

Reply:
Nice! Implemented.

Reply:
That's exactly what I wanted. Thank you for the quick fix and a great PR!